File: python/get_credentials.py
#!/usr/bin/env python3

from _client import get
import os

# Get credentials for a trunk
trunk_sid = os.getenv('TRUNK_SID')
if not trunk_sid:
    print("Error: TRUNK_SID environment variable is required")
    exit(1)

print(f"Getting credentials for trunk {trunk_sid}...")
result = get(f'/trunks/{trunk_sid}/credentials')
print("Credentials retrieved successfully!") 

File: php/get_credentials.php
#!/usr/bin/env php
<?php

require_once '_client.php';

// Get credentials for a trunk
$trunk_sid = getenv('TRUNK_SID');
if (!$trunk_sid) {
    echo "Error: TRUNK_SID environment variable is required\n";
    exit(1);
}

echo "Getting credentials for trunk $trunk_sid...\n";
$result = exo_get("/trunks/$trunk_sid/credentials");
echo "Credentials retrieved successfully!\n";
?> 